lib/kubeclient/watch_stream.rb

```diff
@@ -11,6 +11,7 @@ def initialize(uri, http_options, formatter:)
         @http_client = nil
         @http_options = http_options
         @http_options[:http_max_redirects] ||= Kubeclient::Client::DEFAULT_HTTP_MAX_REDIRECTS
+        @http_options[:keep_alive_timeout] = 60
```

Comment:
this should be like this, this way you can set the value if its being set otherwise it will be default.

@http_options[:keep_alive_timeout] || = Kubeclient::Client::DEFAULT_KEEP_ALIVE_TIMEOUT

Reply:
keep_alive_timeout only set if its passed in otherwise this shouldnt set to keep no changes to existing behavior.